mitiq/benchmarks/random_clifford_t_circuits.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (C) 2020 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Functions for generating randomized benchmarking circuits."""
+from typing import List, Optional, cast
+import copy
+import numpy as np
+
+from cirq.experiments.qubit_characterizations import (
+    _single_qubit_cliffords,
+    _random_single_q_clifford,
+    _random_two_q_clifford,
+    _gate_seq_to_mats,
+    _find_inv_matrix,
+    _two_qubit_clifford_matrices,
+)
+from typing import Sequence
+from cirq import LineQubit, ops,circuits
+from mitiq import QPROGRAM
+from mitiq.interface import convert_from_mitiq
+
+
+def generate_random_clifford_t_circuits(
+    n_qubits: int,
+    num_T: int,
+    num_cliffords: int,
+    trials: int,
+    return_type: Optional[str] = None,
```

Comment:
```suggestion
    n_qubits: int,
    num_oneq_clifford: int,
    num_twoq_clifford: int,
    num_t: int,
    seed: Optional[int] = None,
    return_type: Optional[str] = None,
```
Suggested arguments.  
- Producing a single circuit is enough so `trials` option is not very important, but feel free to keep it. 
- Having a `seed` is important, see e.g. `generate_mirror_circuits()` for an example.
- For this function it is better to set the number of different types of gates instead of the number of clifford group elements.

Reply:
Added `seed` option, added options for different types of gates.